recipes/trust4/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: trust4
+  version: 1.0.2.beta
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/liulab-dfci/TRUST4/archive/v1.0.2-beta.tar.gz
```

Comment:
you can reuse the version variable which you defined above in your URL here with `{{ version }}`

Reply:
The version defined above is 1.0.2.beta since the parser did not accept "-" for the version name.